CHANGELOG

```diff
@@ -7,6 +7,11 @@ This file keeps track of all notable changes to the Slurm Charms.
 Unreleased
 ----------
 
+0.9.3 - 2022-08-18
+------------------
+
```

Comment:
Adding an entry to the Changelog should not change the version of the system. Developing a new feature and carving out a new release are two different processes, let's not mix them together.

```suggestion
```

Reply:
Ok